boards/arm/nucleo_l452re/nucleo_l452re.dts

```diff
@@ -7,6 +7,7 @@
 
 /dts-v1/;
 #include "nucleo_l452re_common.dtsi"
+#include <st/l4/stm32l452r(c-e)tx-pinctrl.dtsi>
 #include "arduino_r3_connector.dtsi"
```

Comment:
I think it is better to do this inclusion is the nucleo_l452re_common.dtsi file as you will reference pinctrl in this file instead of variant (P and non-P). And in fact the common dtsi doesn't include the pinctrl definitions... 🙃

Reply:
Just noticed the 2 inclusions are different... my bad, because revision P has different pinctrl definitions ! I remove my comments !